libs/gl-client/src/lnurl/pay/mod.rs

```diff
@@ -1,25 +1,25 @@
-pub mod models;
-
+use super::models;
+use super::utils::parse_lnurl;
+use crate::lnurl::models::{
+    LnUrlHttpClient, MockLnUrlHttpClient, PayRequestCallbackResponse, PayRequestResponse,
+};
 use anyhow::{anyhow, ensure, Result};
-use bech32::FromBase32;
+use futures::future;
```

Comment:
This import looks unused? Well, it's `future::ready` is being used, but only in the `test` submodule as far as I can see.

Reply:
Moved into the test mod